snappy_pipeline/workflows/ngs_mapping/model.py

```diff
@@ -43,6 +43,9 @@ class TargetCoverageReportEntry(SnappyModel):
       - name: IDT_xGen_V1_0
         pattern: "xGen Exome Research Panel V1\\.0*"
         path: "path/to/targets.bed"
+
+    Bed file for many Agilent exome panels can be found in
+    /fast/work/groups/cubi/projects/biotools/static_data/exome_panel/Agilent
```

Comment:
The existing comment (not the one you added) seem to be incorrect, as I'm quite sure the path will be mapped to the name `"IDT_xGen_V1_0"` not `"default"` ;)
And I'm also not sure whether the pattern actually matches what the description claims.

Reply:
I'm not sure what you mean here: the pattern is a correct (albeit strange) regular expression, and a biomedsheet entry matching such pattern is associated with the name, which is finally mapped to a path.